test/e2e/util/utils.go

```diff
@@ -764,25 +763,21 @@ func DeployEraserManifest(namespace, fileName string) env.Func {
 	}
 }
 
-func CreateExclusionList(namespace string, list pkgUtil.ExclusionList) env.Func {
+func CreateExclusionList(namespace string, list string) env.Func {
 	return func(ctx context.Context, cfg *envconf.Config) (context.Context, error) {
 		c, err := cfg.NewClient()
 		if err != nil {
 			return ctx, err
 		}
 
-		b, err := json.Marshal(&list)
-		if err != nil {
-			return ctx, err
-		}
-
 		// create excluded configmap and add docker.io/library/alpine
 		excluded := corev1.ConfigMap{
 			ObjectMeta: metav1.ObjectMeta{
 				GenerateName: "excluded",
 				Namespace:    EraserNamespace,
+				Labels:       map[string]string{"eraser.sh/exclude.list": "true"},
```

Comment:
how was this working before?

Reply:
the test wasn't waiting for the collector pods to finish, so when it checked that the images were still on the node it would return true right away